core/src/data/row.rs

```diff
@@ -118,7 +117,7 @@ impl Row {
             } = column_def;
 
             value.validate_type(data_type)?;
-            value.validate_null(nullable)?;
+            value.validate_null(nullable.to_owned())?;
```

Comment:
`to_owned()` for `Copy` impl value might be.. too much.
you can simply use `*nullable` for these cases.